Backend/DataAccessLayer/Repositories/SessionRepository.cs

```diff
@@ -63,5 +71,12 @@ public Session ExpireSession(DatabaseContext _db, string token)
             _db.Entry(session).State = EntityState.Modified;
             return session;
         }
+
+        public Guid GetSessionGuid(DatabaseContext _db, string token)

```

Comment:
No need to retrieve the Session Guid with a separate function. It is already accessible in the object.